mkchimeenv.py

```diff
@@ -18,18 +18,17 @@
 from virtualenvapi.manage import VirtualEnvironment
 
 
-__version__ = "2022.06"
+__version__ = "2023.05"
 
 
-chime_repositories = {
+public_repositories = {
```

Comment:
Good idea! I've added this in a slightly ugly way. If you have suggestions for an improvement, let me know; otherwise, I think it's ready to be merged

Reply:
It doesn't look that ugly to me! Maybe my sense of code aesthetics is diminishing.